detector/aws_instance_not_specified_iam_profile.go

```diff
@@ -17,3 +25,7 @@ func (d *Detector) DetectAwsInstanceNotSpecifiedIamProfile(issues *[]*issue.Issu
 		}
 	}
 }
+
+func (d *AwsInstanceNotSpecifiedIAMProfileDetector) Inherit(original *AwsInstanceNotSpecifiedIAMProfileDetector) *AwsInstanceNotSpecifiedIAMProfileDetector {
```

Comment:
__[gometalinter]__
* [golint] exported method AwsInstanceNotSpecifiedIAMProfileDetector.Inherit should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->